app/views/components/docs/notice.yml

```diff
@@ -9,6 +9,8 @@ accessibility_criteria: |
 
   - have a border colour contrast ratio of more than 4.5:1 with its background to be visually distinct.
   - always render headings with associated description content, so there are no isolated heading elements inside the component
+shared_accessibility_criteria:
+  - link
```

Comment:
Should this be included in this component? The notice component itself doesn't contain any links (as far as I know)?

Reply:
Yeah, I wasn't sure - but one of the examples in the guide has links in. They don't get added directly, I guess. I might be jumping the gun as this might fit more into the bracket of accessibility guidance for inputs.

Reply:
I think if we add link accessibility criteria for all components that may, at some point, include a link being passed to them in some HTML, this might mean we have to add it to the majority of components, like banner, lead para etc... To me, that doesn't seem right, but that's just my personal opinion 🙂

Reply:
I agree with @vanitabarrett